### ISSUE TITLE
No way to go to back to home page when inside of a project

### ISSUE THREAD

**Issue by rishijain** [fastruby/points#215]
When I go inside a project, the link at the left top (in same bar as the sign out button) changes to current project.
So clicking on it, brings me back to same project page.

While for coming back to same project (in case user is inside the sub project), there is a link to navigate back to the main project.
The only way to go back to the root page or the list of all projects page is by directly changing the url in the browser.

I am not sure if this is an intended behaviour and hence not opening as a bug or as a feature.

![s1](https://user-images.githubusercontent.com/946527/155140443-fe750e14-e8f8-4d61-839b-aa8416d28cd7.png)

**Comment by arielj** [comment]
There's the `Return to Projects Page` button at the bottom right of the list of buttons (really bad place though haha).

**Comment by rishijain** [comment]
ahaaaa. yes now I see it. 

With all those options related to that specific project, I missed this button.

**Comment by arielj** [comment]
yeah, it's in a really bad place, it was discussed a few times but we never fixed it, I understand the idea is to put something next to the project's name in the navbar, but we never decide what exactly

before, the navbar only said `Points` with a link to the index, but it was replaced with the project name because in many pages there was no context of the current project

**Comment by rishijain** [comment]
should we put it in the navbar just before the "project name" and call it "Home"? 
Something like this
![Screenshot from 2022-02-22 19-28-02](https://user-images.githubusercontent.com/946527/155147012-6d0eb11e-79dc-4b83-971e-6b63f753c954.png)

**Comment by arielj** [comment]
maybe? haha, it was never defined, you can ask in the ux-ui channel in slack to see what people think and nathalie will probably try some ideas, not sure what's the best UX there
